### PR TITLE
CB-17407 Install postgresql 11 next to 10 on the images

### PR DESCRIPTION
**Pull request by lacikaaa**
The default is still postgresql 10. Both databases are initialized.
Tested with creating AWS and docker image.

### CODE REVIEW

File: saltstack/base/salt/postgresql/init.sls [hunk 1]
```diff
@@ -20,6 +28,121 @@ install-postgres:
       - postgresql10-docs
       - postgresql10-devel
 
+
+install-centos-sclo-rh:
```
Comment (Bajzathd) [inline saltstack/base/salt/postgresql/init.sls:32]: typo

File: saltstack/base/salt/postgresql/init.sls [hunk 2]
```diff
@@ -20,6 +28,121 @@ install-postgres:
...
+  cmd.run:
+    - name: yum -y install centos-release-scl-rh
```
Comment (Bajzathd) [inline saltstack/base/salt/postgresql/init.sls:34]: I think this should be replaced with pkg.installed

Comment (lacikaaa) [reply saltstack/base/salt/postgresql/init.sls:34]: right, let me try it out first

**Comment by Bajzathd** [comment]
Do we need to initialize and run both? In case of java11 only the package was installed, and the initialization was done with recipes for testing. Maybe the same could be done here @pkedvessy ?
Also, could this cause any performance issues to have 2 dbs running?

**Review by Bajzathd — APPROVED** [review]

**Pull request merged**